graylog2-web-interface/src/components/graylog/SplitButton.jsx

```diff
@@ -10,7 +10,7 @@ const SplitButton = forwardRef(({ active, bsStyle, ...props }, ref) => {
     ${buttonStyles({ active })};
 
     ~ .btn.dropdown-toggle {
-      ${buttonStyles({ active, specific: false })};
```

Comment:
So far this is the only part where I am not totally sure what the effects are: I think we did not had to use the specificity here, because `~ . btn.dropdown-toggle` is increasing it anyway. In this case we can safely do this change.